tests/ObjectTest.php

```diff
@@ -69,4 +69,16 @@ public function testRecursiveToArray()
         $this->assertArrayHasKey('foo', $converted['child']);
         $this->assertEquals('a', $converted['child']['foo']);
     }
+
+    public function testInexistentProperty()
```

Comment:
Sorry for the super nit, but can we call this `testNonexistentProperty` or `testPropertyDoesNotExists`. `Inexistent` is valid english, but is fairly rarely used.